UPGRADE.md

```diff
@@ -0,0 +1,65 @@
+# Upgrade Guide
+
+## 2.x -> 3.0.0
+
+When upgrading from 2.x to 3.0.0 of OS2Display, a mayor change has been introduced to the project.
```

Comment:
Upgrading OS2Display from version 2.x to 3.0.0 introduces a major change to the project.